wbooks/src/app/screens/booklist/index.js

```diff
@@ -1,15 +1,35 @@
 import React, { Component } from 'react';
-import { View, StyleSheet } from 'react-native';
+import { 
+	SafeAreaView, 
+	View, 
+	StyleSheet, 
+	FlatList 
+} from 'react-native';
 import BookItem from './components/BookItem';
 import styles from './styles';
+import books from './books';
```

Comment:
if books is a constant it should be in capital letters like `BOOKS`, if it is a component it should be `Books`.

Reply:
ok, BOOKS then